packages/codec/src/base.ts

```diff
@@ -114,3 +114,90 @@ export function createFixedBytesCodec<Unpacked, Packable = Unpacked>(
     }),
   };
 }
+
+const LAST_ACCESS_PATH = Symbol("PACKABLE_ACCESS_PATH");
```

Comment:
as a lib, Lumos should have as minimal dependencies and state as possible

we should avoid using symbol, because users may use multiple versions of the codec, which will make the old codec and the new version not work properly

for example, if a library uses `codec@0.1` to create a blockchain.mol codec and the user uses `codec@0.2`, this will make `codec0_2HighOrderCodec(codec0_1.someCodec)` not work properly because they are not using the same symbol